wizard/src/utils/detect_place.ts

```diff
@@ -28,52 +67,78 @@ export class PlaceDetector {
   countryAbbrv3: Set<string>;
   countryNumeric: Set<string>;
 
-  placeTypes: Map<string, string>;
+  // Convenience in-memory maps of types and properties where the keys are their
+  // respective DC Names.
+  placeTypes: Map<string, DCType>;
+  placeProperties: Map<string, DCProperty>;
 
-  static typeFormatMappings = new Map<string, Array<DetectedFormat>>([
-    ["Longitude", [{ propertyName: "longitude", displayName: "Longitude" }]],
-    ["Latitude", [{ propertyName: "latitude", displayName: "Latitude" }]],
-    [
-      "LatLon",
-      [{ propertyName: "GeoCoordinates", displayName: "Geo Coordinates" }],
-    ],
-    [
-      "GeoCoordinates",
-      [{ propertyName: "GeoCoordinates", displayName: "Geo Coordinates" }],
-    ],
+  // A set of all place types and their associated properties.
+  placeTypesAndProperties: Set<TypeProperty>;
+
+  // Mapping between Place types and supported properties associated with each
+  // type. The keys of keyTypePropertyMappings are matched against the column
+  // headers in the user csv files. If a column header matches a key in
+  // keyTypePropertyMappings then the associated value in
+  // keyTypePropertyMappings is the inferred location type and property.
+  static keyTypePropertyMappings = new Map<string, Array<TPName>>([
```

Comment:
nit: would columnToTypePropertyMap clarify it further... (feel free to ignore/defer)

Reply:
Done!